plugin.json

```diff
@@ -22,42 +22,42 @@
         "settings": [
             {
                 "key": "JitsiURL",
-                "display_name": "Jitsi Server URL",
+                "display_name": "Jitsi URL",
                 "type": "text",
                 "help_text": "The URL for an on-premise Jitsi server. For example, https://jitsi.example.com.",
                 "placeholder": "https://jitsi.example.com"
             },
             {
                 "key": "JitsiJWT",
-                "display_name": "Enable Jitsi JWT",
+                "display_name": "Use JWT Authentication for Jitsi",
                 "type": "bool",
-                "help_text": "Enable this, if You have a Jitsi with JWT"
+                "help_text": "(Optional) If your Jitsi server uses JSON Web Tokens (JWT) for authentication, set this value to true."
             },
             {
                 "key": "JitsiAppID",
-                "display_name": "Jitsi Server app ID",
+                "display_name": "App ID for JWT Authentication",
                 "type": "text",
-                "help_text": "The App ID on Your Jitsi JWT server"
+                "help_text": "(Optional) The App ID used for authentication by the Jitsi server and JWT token generator."
             },
             {
                 "key": "JitsiAppSecret",
-                "display_name": "Jitsi Server app secret",
+                "display_name": "App Secret for JWT Authentication",
                 "type": "text",
-                "help_text": "The App secret on Your Jitsi JWT server"
+                "help_text": "(Optional) The App Secret used for authentication by the Jitsi server and JWT token generator."
             },
             {
                 "key": "JitsiLinkValidTime",
-                "display_name": "Link valid time, in minutes",
+                "display_name": "Meeting Link Expiry Time (minutes)",
                 "type": "number",
-                "help_text": "How many minutes is the generated link active, if set smaller then 1, valid time is automaticaly set to 30 minutes.",
+                "help_text": "(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute.",
                 "default": 30
             },
             {
                 "key": "JitsiNamingScheme",
-                "display_name": "Meeting naming scheme",
+                "display_name": "Meeting Naming Scheme",
```

Comment:
@jespino I'm curious if we should reduce the number of options to three: `english-titlecase`, `UUID` and `teamchannel`. I can see each being useful (the first to avoid breaking changes; second for uniqueness; third based on user feedback) - but I'm wondering how useful the other options are if these three are available.

@rbdevr @w00dst0ck @lediur feedback welcome, since you've previously proposed changes to the meeting naming schema. cc @seansackowitz of course as well :)

Reply:
I put in multiple options to reflect how people might use it or share links with others outside of Mattermost. For some teams, a short numeric meeting ID is easier to type if you're joining via the website or the mobile app, so having those options seemed painless to add.

Reply:
I think we can reduce the number of options, and actually, I think we can have a "mattermost-style" one, something like "in direct/group messages use a personal room (username-sha1(user_id))" and for channels use "channel_name-10-random-characters" or something like that

Reply:
I would add the "mattermost-style", the one proposed before. and I would remove english-kebabcase, teamchannel and teamchannel-salt, and maybe letters

Reply:
Thanks all! Will see if others have feedback/thoughts